openfecwebapp/templates/partials/site-orientation.html

```diff
@@ -1,16 +1,28 @@
 <div class="js-new-site-orientation site-orientation">
   <div class="banner" role="banner" aria-hidden="false">
     <div class="container">
-      <span class="t-bold">In May, this site will become the new FEC.gov</span>
-      <div class="toggle" aria-hidden="false">
-        <span class="t-block">After that, the <a href="http://www.fec.gov">current FEC.gov</a> will be archived and made available at classic.fec.gov</span>
-        <span class="t-block">Questions or comments? Use our <a class="js-feedback">anonymous feedback tool</a> or email <a href="mailto:webmanager@fec.gov">webmanager@fec.gov</a></span>
-        <div class="tour__start"><button class="tour__start__button">Start a tour</button> of the new features</div>
-      </div>
-      <div class="toggle-text">
-        <button class="less" aria-hidden="false" tabindex="0">Show less <i class="icon-small--arrow-up-border"></i></button>
-        <button class="more" aria-hidden="true" tabindex="0">Show more <i class="icon-small--arrow-down-border"></i></button>
-      </div>
+      {% if canonical_base == 'https://fec.gov' %}
+        <span class="t-bold">We’ve updated FEC.gov</span> | You can still access an archive of the old FEC.gov at <a href="http://classic.fec.gov">classic.fec.gov</a>
+        <div class="toggle" aria-hidden="false">
+          <div class="tour__start"><button class="tour__start__button">Start a tour</button> of the new features</div>
+          <span class="t-block">Questions or comments? Use our <a class="js-feedback" href="javascript:void(0)">anonymous feedback tool</a> or email <a href="mailto:webmanager@fec.gov">webmanager@fec.gov</a></span>
```

Comment:
Ah great. Should I pull this out or you can just fix it when resolving conflicts?